phoenix-protocol/src/main/PGuidePosts.proto

```diff
@@ -24,7 +24,9 @@ option java_generic_services = true;
 option java_generate_equals_and_hash = true;
 option optimize_for = SPEED;
 message PGuidePosts {
-  repeated bytes guidePosts = 1;
```

Comment:
These fields were optional and will not impact as we are not using stats with old client. But anyways I have made the change to add a new field and keep the old one as it is.